core/templates/dev/head/collection_player/collection_player.html

```diff
@@ -0,0 +1,89 @@
+{% extends "base.html" %}
+
+{% block subtitle %}
+  {{ collection_title }}
+{% endblock subtitle %}
+
+{% block header_js %}
+  {{ super() }}
+  {% if collection_id %}
+    <script type="text/javascript">
+      GLOBALS.collectionId = JSON.parse('{{collection_id|js_string}}');
+      GLOBALS.isLoggedIn = Boolean(JSON.parse('{{is_logged_in|js_string}}'));
+    </script>
+
+    <style>
+      html, body {
+        background-color: #eee;
+      }
+    </style>
+  {% endif %}
+
+  {{dependencies_html}}
+{% endblock header_js %}
+
+{% block navbar_breadcrumb %}
+  <ul class="nav navbar-nav oppia-navbar-breadcrumb">
+    <li>
+      <span class="oppia-navbar-breadcrumb-separator"></span>
+      <span class="protractor-test-collection-header" itemprop="description">{{collection_title}}</span>
+    </li>
+  </ul>
+{% endblock navbar_breadcrumb %}
+
+{% block content %}
+  <div ng-controller="CollectionPlayer">
+    <div ng-if="collection" class="oppia-collection-player-tiles-area">
+      <div class="oppia-collection-player-tiles-container">
+        <div ng-if="!hasFinishedCollection()" class="oppia-collection-player-tiles-header">
+          <span ng-if="!hasStartedCollection()">Begin by playing:</span>
+          <span ng-if="hasStartedCollection()">Continue where you left off:</span>
+        </div>
+        <div ng-if="hasFinishedCollection()" class="oppia-collection-player-tiles-header">
+          <span>You have finished the collection! Feel free to replay any explorations below.</span>
+        </div>
+
+        {% if not is_logged_in %}
+          <span class="oppia-collection-player-small-text">You must login to save your progress.</span>
+        {% endif %}
+
+        <div ng-repeat="collection_node in getDisplayedCollectionNodes()">
+          <md-card layout="row" class="oppia-collection-player-tile protractor-test-collection-player-tile">
+            <a ng-href="/explore/<[collection_node.exploration.id]>?collection_id=<[collectionId]>" class="oppia-collection-player-tile-container-link" style="text-decoration: none;">
+              <div ng-class="{'oppia-collection-player-finished-exploration-tile': collection_node.exploration.completed}" class="oppia-collection-player-tile-contents">
+                <span class="oppia-collection-player-tile-image-container">
+                  <img class="oppia-collection-player-tile-image" ng-src="/images/gallery/exploration_background_teal_small.png">
+                </span>
+
+                <div class="oppia-collection-player-tile-details">
+                  <span class="oppia-collection-player-tile-title protractor-test-gallery-tile-title">
+                    <[collection_node.exploration.title]>
```

Comment:
I'm actually hesitant on both points, though I do agree they are empty. Here are my thoughts:

1. Regarding listing the details of explorations such as on the gallery page, I think most of this information is actually quite noisy for a collection. For instance, I think seeing 'last updated' is going to be irrelevant to the experience a learner should have when playing through a collection.  The category is also irrelevant since most of the explorations will be in the same category as the collection and, thus, adds nothing to the learning experience. Finally, the objective of the exploration might actually conflict or be different from the information being taught in the collection. This is outside the power of the collection author to deal with, so I'd rather not display it in the collection page if they can't edit it.

That being said, we should definitely figure out what sort of information would be good to display on the card and ensure it can be changed by the collection author.

1. Regarding displaying pre-requisite (and possibly acquired) skills, the original intention of skills was meant to be hidden from the learner (at least in the MVP). I think it's somewhat important for learners to know where they are going with a topic, but I think it's less important for them to know the skills in order to reach this. It probably would be more effective to let them see the skills if they manually enable them and instead require or encourage the collection author to describe to the learners the purposes of an exploration and briefly what it will teach.

Overall, I seem to be suggesting:

1. We allow collection authors to define custom exploration objectives per exploration in their collection. This is displayed on the card of the exploration in the collection learner view.

1. We allow collection authors to describe the purpose/objective of each skill, then list this information on the card for acquired skills (this might actually be better than a manual objective statement).

1. We *may* also want to enumerate the list of pre-requisite skills, but this could be done via the sentence objectives of each skill the collection author defines.

These are early thoughts and definitely have some initial problems (such as too many prereq/acquired skills to display sentences for or objectives which are too long to cleanly display). We should definitely discuss this at greater length.

WDYT (+ @seanlip)?